app/models/manual_record.rb

```diff
@@ -12,7 +12,7 @@ class ManualRecord
     autosave: true
 
   def self.find_by(attributes)
-    first(conditions: attributes)
+    where(attributes).first
```

Comment:
If I'm understanding the documentation correctly, I think we might be able to remove this method entirely, because it's [implemented in Mongoid](http://www.rubydoc.info/gems/mongoid/3.0.0/Mongoid/Finders#find_by-instance_method).